android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -328,6 +348,12 @@ class XMTPModule : Module() {
             ) ?: return
         subscriptions[conversation.cacheKey(clientAddress)]?.cancel()
     }
+
+    private fun logV(msg: String) {
+        if (isDebugEnabled) {
+            Log.v("XMTPModule", msg);
+        }
```

Comment:
added this in response to https://github.com/xmtp/xmtp-react-native/pull/45#pullrequestreview-1447012410